### ISSUE TITLE
OperationalError: cannot commit with sqlite

### ISSUE THREAD

**Issue by jandot** [catherinedevlin/ipython-sql#30]
Hi,

A colleague and I have been trying to get ipython-sql working with sqlite, following the example from http://nbviewer.ipython.org/gist/catherinedevlin/5272681. Unfortunately, we keep getting the following error: "(OperationalError) cannot commit - no transaction is active 'commit' ()". The extension does work for mysql in our case, e.g. " mysql://anonymous ensembldb.ensembl.org/some-database".

It seems that a table creation itself works when we check that in the sqlite console, but otherwise we get the error.

Any workaround or fix much appreciated.
Jan. (vda-lab.be)

**Comment by stonebig** [comment]
Same issue here.
You may try to path Lib\site-packages\sql\run.py , line 274

before:
````
            conn.session.execute('commit')
````

after:
````
            try:
                conn.session.execute('commit')
            except:
                pass
````

![ipython_sql_issue_on_sqlite](https://cloud.githubusercontent.com/assets/4312421/6098780/22018b90-afe9-11e4-90e9-e4378607a040.GIF)

**Comment by jandot** [comment]
That works. Thanks!

jan.

**Comment by alexandrul** [comment]
I've got the same error with 0.35, and also other errors with mssql+pyodbc.

Reverting to 0.34 solved both issues in my case.

**Comment by catherinedevlin** [comment]
Sorry about that!  Thanks @jandot and @stonebig, fix is in 0.3.6.
